src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Todo, User } from '../../types';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  onAdd: (newTodo: Todo) => void;
+  users: User[];
+};
+
+export const TodoForm: React.FC<Props> = ({ onAdd }) => {
```

Comment:
The `users` prop is defined in the `Props` type but is not used in the component. Make sure to destructure `users` from the props and use it in the component instead of `usersFromServer`.